partner_delivery_schedule/models/partner_delivery_schedule.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class PartnerDeliverySchedule(models.Model):
+    _name = 'partner.delivery.schedule'
+
+    name = fields.Char(compute='_compute_name', store=True)
+    partner_id = fields.Many2one(
```

Comment:
This shouldn't have `partner_id`. In my POV, you define generic delivery interval: "from 10:00 to 14.00", and then you assign that delivery interval to the partners you want. It should also include optional days (multiple selection), as on Mondays can be from 10:00 to 14:00, but on Tuesdays and Fridays from 9:00 to 13.00, and no delivery on Saturdays nor Sundays. The `name_get` can be composed with initial letters of each day (ordered) + delivery interval